Apps/phgsgmail/gsgmail.json

```diff
@@ -791,12 +851,54 @@
                     ]
                 }
             ],
+            "render": {
+                "width": 12,
+                "title": "Delete Email",
+                "type": "table",
+                "height": 5
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "on poll",
+            "description": "Callback action for the on-poll ingest functionality",
+            "verbose": "Please see sections <a href=\"#poll_now\">POLL NOW</a> and <a href=\"#scheduled_polling\">Scheduled Polling</a> for more info on how this action is implemented by the app.",
+            "type": "ingest",
+            "identifier": "on_poll",
+            "read_only": true,
+            "parameters": {
+                "start_time": {
+                    "data_type": "numeric",
+                    "description": "Parameter Ignored in this app"
+                },
+                "end_time": {
+                    "data_type": "numeric",
+                    "description": "Parameter Ignored in this app"
+                },
+                "container_id": {
+                    "data_type": "string",
+                    "description": "Parameter Ignored in this app"
+                },
+                "container_count": {
+                    "data_type": "numeric",
+                    "description": "Maximum number of emails to ingest",
+                    "required": true,
+                    "value": 100
+                },
+                "artifact_count": {
+                    "data_type": "numeric",
+                    "description": "Maximum number of artifact to ingest",
+                    "required": true,
+                    "value": 1000
+                }
+            },
+            "output": [],
             "versions": "EQ(*)"
         },
         {
             "action": "get email",
             "description": "Retrieve email details via internet message id",
-            "verbose": "Action uses the GMail API to search in a user's mailbox (specified in the <b>email</b> parameter). Use the <b>run query</b> action to retrieve <b>internet message id</b>.<br>Use <b>extract attachments</b> parameter to add attachments to vault and add corresponding vault artifacts.<br>Requires authorization with the following scope:<ul><li>https://www.googleapis.com/auth/gmail.readonly</li></ul>",
+            "verbose": "Action uses the GMail API to search in a user's mailbox (specified in the <b>email</b> parameter). Use the <b>run query</b> action to retrieve <b>internet message id</b>.<br>Use <b>extract attachments</b> parameter to add attachments to vault and add corresponding vault artifacts.<br>Requires authorization with the following scope:<ul><li>https://www.googleapis.com/auth/gmail.readonly</li></ul>.",
```

Comment:
This .(period) is reflecting in new line
<img width="780" alt="Screenshot 2021-10-04 at 3 25 31 PM" src="https://user-images.githubusercontent.com/64279254/135831248-15ee4c9d-f999-4e16-9e09-b76d3903d24c.png">